core/sched/Makefile

```diff
@@ -1,6 +1,9 @@
 objs-y :=
 
 objs-y += scheduler.o
+objs-y += sched-config.o
+
 objs-y += rr.o
+objs-y += rt-rm.o
```

Comment:
It could be good if we can use menuconfig,

But unfortunately, as the number of pCPU is not fixed, it seems we can not use menuconfig for this.